rollup.config.js

```diff
@@ -1,59 +1,40 @@
-
-import builtins from 'rollup-plugin-node-builtins'
-import replace from 'rollup-plugin-re'
 import resolve from 'rollup-plugin-node-resolve'
 import commonjs from 'rollup-plugin-commonjs'
-import json from 'rollup-plugin-json'
+import babel from 'rollup-plugin-babel'
+import pkg from './package.json'
 
 export default [{
     input: 'src/index.js',
     name: 'ocapi-client',
     output: {
-        file: 'dist/commercecloud-ocapi-client.js',
+        name: 'OcapiClient',
+        file: pkg.browser.file,
         format: 'umd'
     },
     plugins: [
-        json(),
-        /*
-         * NOTE: Do replace before commonjs
-         * This is a hack for rollup
-         * In current rollupJS version, it does not handle reassigned require function
-         * in certain packages that uses commonjs helper
-         * We need to manually remove the hijacked require function
-         * For more info: https://github.com/rollup/rollup-plugin-commonjs/issues/166#issuecomment-328853157
-         */
-        replace({
-            patterns: [{
-                match: /formidable(\/|\\)lib/,
-                test: 'if (global.GENTLY) require = GENTLY.hijack(require);',
-                replace: '',
-            }]
-        }),
         commonjs({
             namedExports: {
-                superagent: ['superagent'],
+                './node_modules/superagent/superagent.js': ['superagent'],
             },
         }),
         resolve({
             preferBuiltins: false,
             querystring: true,
+            browser: true,
         }),
-        builtins(),
+        babel({
+            exclude: ['node_modules/**']
+        })
     ]
 },
 {
     input: 'src/index.js',
-    output: {
-        file: 'lib/index.js',
+    output: [{
+        file: pkg.main,
         format: 'cjs'
```

Comment:
Do we also want to add babel to cjs build?

Reply:
Yep.  Added that back in.  Now everything builds into `/dist` so that it's easier to know where everything is.